src/Toggle/Toggle.styles.ts

```diff
@@ -0,0 +1,67 @@
+import styled from 'styled-components';
+import { Theme } from '../shared/theme.types';
+
+interface Props {
+  size: 'small' | 'medium';
+}
+
+export const ToggleStyled = styled.div<Props>`
+  width: ${({ size }) => (size === 'medium' ? '40px' : '28px')};
+  height: ${({ size }) => (size === 'medium' ? '24px' : '16px')};
+  position: relative;
+  display: inline-block;
+
+  input {
+    opacity: 0;
+    z-index: 2;
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
```

Comment:
```suggestion
    bottom: 0;
    cursor: pointer;
```

Cursor pointer is only visible in the border of Toogle, perhaps we can add it here.

Reply:
🤦 It completely escaped when fixing something else. Very good catch.